### PR TITLE
Multipart

### PR DESCRIPTION
**Pull request by cy6erGn0m**
Just a preview, don't merge it. Depends on PR #10  and PR #11 so ignore first commits

### CODE REVIEW

**Comment by orangy** [comment]
I don't see any tests.

**Comment by cy6erGn0m** [comment]
We need integration tests for every container. The only I can test for now is `ContentDisposition` rendering  (done)

File: ktor-samples/ktor-samples-post/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+ktor {
+    deployment {
+        environment = development
+        port = 8080
+    }
+
+    application {
+        package = org.jetbrains.ktor.samples.locations
```
Comment (orangy) [inline ktor-samples/ktor-samples-post/resources/application.conf]: package name :)

File: ktor-samples/ktor-samples-post/web/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+         version="3.0">
+    <servlet>
+        <display-name>Ktor Locations Sample</display-name>
```
Comment (orangy) [inline ktor-samples/ktor-samples-post/web/WEB-INF/web.xml]: Post Sample

**Pull request merged**